dashboard/dashboard.py

```diff
@@ -242,17 +252,39 @@ def analyze_experiment(info, experiments_dir, tmp_data_dir,
 
     status = validate_status(tmp_analysis_dir)
 
+    # collect data to dump to data_*.json
+    dump_data = {}
     # read the analyzed data, append a timestamp field, and copy over to the permanent data dir
     if status['success']:
         data_exists = check_file_exists(tmp_analysis_dir, 'data.json')
         if not data_exists:
             status = {'success': False, 'message': 'No data.json file produced by {}'.format(exp_name)}
         else:
             data = read_json(tmp_analysis_dir, 'data.json')
-            data['timestamp'] = date_str
-            data['tvm_hash'] = tvm_hash
-            write_json(analyzed_data_dir, 'data_{}.json'.format(date_str), data)
-
+            data.update({
```

Comment:
Nitpick: Since you're copying the data into `dump_data`, you could probably put `timestamp` and `tvm_hash` directly into `dump_data` instead of modifying `data`